### ISSUE TITLE
Accessibility: Can't access text fields with TalkBack screen reader.

### ISSUE THREAD

**Issue by shortwavesurfer2009** [Kunzisoft/KeePassDX#2336]
### Checks

- [x] I have read the Wiki, searched the open issues, and still think this is a new bug.

### Explain the problem clearly and succinctly:

When using the Talkback screen reader on Android, I am not able to access fields such as title, URL, notes, etc.

### Describe what you expected to happen:

Talkback users should be able to access the edit fields for title, URL, notes, etc.

### KeePassDX version:

4.3.0

### Build:

Libre

### Database version:

4

### Android version:

15

### Android device:

OnePlus Nord N200

**Comment by J-Jamet** [comment]
I am aware that there is still much room for improvement in terms of accessibility.
I was planning to do a complete overhaul with the switch to Compose, but if the changes are not too time-consuming, I will see if I can improve the current dynamic views.
